<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogoActivoFijoGUI</class>
 <widget class="QWidget" name="CatalogoActivoFijoGUI">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Catalogo de activo fijo</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>871</width>
     <height>441</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Activo Fijo</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Cuenta de inventario y clase</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Cuenta:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Clase:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombreClase">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>95</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="cuentaAsignada">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="numeroCuenta">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="descripcion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>841</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="botonEliminar">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Eliminar</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/editdelete.png</normaloff>:/iconos/editdelete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonNuevo">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Nuevo</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonModificar">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Modificar</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/trabajador.png</normaloff>:/iconos/trabajador.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonGuardar">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Guardar</string>
     </property>
     <property name="whatsThis">
      <string extracomment="Guardar"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="ayudaActivoFijo">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QTreeWidget" name="catalogoInventario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>256</width>
       <height>341</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Catalogo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Codigo</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>200</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Saldo Deudor</string>
     </property>
     <widget class="QLineEdit" name="saldoDeudor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>110</y>
       <width>139</width>
       <height>79</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Debe</string>
     </property>
     <widget class="QLineEdit" name="debe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>110</y>
       <width>139</width>
       <height>79</height>
      </rect>
     </property>
     <property name="title">
      <string>Haber</string>
     </property>
     <widget class="QLineEdit" name="haber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>200</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Saldo Acreedor</string>
     </property>
     <widget class="QLineEdit" name="saldoAcreedor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>270</y>
       <width>279</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Descripción</string>
     </property>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>279</width>
       <height>49</height>
      </rect>
     </property>
     <property name="title">
      <string>Nombre de la Cuenta</string>
     </property>
     <widget class="QLineEdit" name="codigoCuenta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btmSeleccionar">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/button_ok.png</normaloff>:/iconos/button_ok.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/exit.png</normaloff>:/iconos/exit.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btmBuscar">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>330</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
     <property name="icon">
      <iconset resource="../recursosIconos.qrc">
       <normaloff>:/Iconos/iconos/buscar.png</normaloff>:/Iconos/iconos/buscar.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Material Bibliografico</string>
    </attribute>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>561</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingreso de especificaciones de Material Bibliografico</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Ingrese la clase: </string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cuenta:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Clase:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Especifico:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="claseBiblio">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="cuentaBiblio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="especificoBiblio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Datos Generales autor:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>551</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Datos Generales</string>
     </property>
    </widget>
    <widget class="QToolButton" name="botonEliminarBiblio">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/editdelete.png</normaloff>:/iconos/editdelete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonNuevoBiblio">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonModificaBiblio">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/trabajador.png</normaloff>:/iconos/trabajador.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonGuardarBiblio">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Descripcion de la Clase:</string>
     </property>
     <widget class="QLineEdit" name="descripcionBiblio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>511</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Unidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Departamento:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreUnidad">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreDpto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../ContablesPrincipal.qrc">:/iconos/kedit.png</pixmap>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Cree, modifique o elimine especificaciones de los bienes inventariables.</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventario:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="anioInventario">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>numeroCuenta</tabstop>
  <tabstop>nombreClase</tabstop>
  <tabstop>botonGuardar</tabstop>
  <tabstop>botonModificar</tabstop>
  <tabstop>botonNuevo</tabstop>
  <tabstop>botonEliminar</tabstop>
  <tabstop>ayudaActivoFijo</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>claseBiblio</tabstop>
  <tabstop>descripcionBiblio</tabstop>
  <tabstop>botonGuardarBiblio</tabstop>
  <tabstop>botonModificaBiblio</tabstop>
  <tabstop>botonNuevoBiblio</tabstop>
  <tabstop>botonEliminarBiblio</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>especificoBiblio</tabstop>
  <tabstop>cuentaBiblio</tabstop>
  <tabstop>nombreUnidad</tabstop>
  <tabstop>nombreDpto</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>cuentaAsignada</tabstop>
 </tabstops>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
  <include location="../recursosIconos.qrc"/>
 </resources>
 <connections/>
</ui>
